sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionEventArgs.cs

```diff
@@ -31,6 +31,11 @@ public class ProcessSessionEventArgs : EventArgs
         /// </summary>
         public string SessionId => _sessionReceiver.SessionId;
 
+        /// <summary>
+        /// The identifier of the Service Bus entity that the message was received from.
+        /// </summary>
+        public string Identifier => _sessionReceiver.Identifier;
```

Comment:
Use processor identifier and add new ctor. We might consider just taking the Processor in the new ctor rather than the Identifier so we don't have to keep adding new ctors in the future.